CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+# Latest updates
+
+From v1.6.6, You can check all of our changes from [Releae Page](https://github.com/Ikuyadeu/vscode-R/releases)
```

Comment:
typo `Releae`?

Reply:
Yes, I should add the typo checker in the future.